pkg/reporting/nodes.go

```diff
@@ -6,33 +6,107 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/chef/chef-analyze/pkg/config"
+	chef "github.com/chef/go-chef"
 )
 
-func Nodes(cfg *config.Config) error {
+type CookbookVersion struct {
+	Name    string
+	Version string
+}
+
+func (cbv CookbookVersion) String() string {
```

Comment:
We should define these functions on pointers to the object so we're not passing around the whole object all the time

Reply:
I've opened https://github.com/chef/chef-analyze/issues/60 for this.

Reply:
I think the only thing you need to do is add a `*` to the instance of the struct.
```suggestion
func (cbv *CookbookVersion) String() string {
```